packages/bitcore-wallet-client/src/lib/api.ts

```diff
@@ -2386,6 +2429,22 @@ export class API extends EventEmitter {
               clients.push(tokenClient);
             });
           }
+          // Eth wallet with mulsig wallets?
+          const multisigEthInfo = status.preferences.multisigEthInfo;
+          if (!_.isEmpty(multisigEthInfo)) {
+            _.each(multisigEthInfo, info => {
+              log.info('Importing multisig wallet');
```

Comment:
add ` info.multisigContractAddress` to the log

Reply:
Done. Thanks !